pom.xml

```diff
@@ -69,10 +69,13 @@
             <artifactId>h2</artifactId>
             <version>1.4.186</version>
         </dependency>
-                
-                
-
-        <!--test scope-->
+		<dependency>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Tab characters found (this is the first occurrence) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=xml%3AIllegalTabCheck)